gax-java/gax/src/main/java/com/google/api/gax/tracing/MetricsTracer.java

```diff
@@ -42,12 +52,173 @@
 @InternalApi
 public class MetricsTracer implements ApiTracer {
 
-  public MetricsTracer(MethodName methodName, MetricsRecorder metricsRecorder) {}
+  private static final String STATUS_ATTRIBUTE = "status";
+
+  private Stopwatch attemptTimer;
+
+  private final Stopwatch operationTimer = Stopwatch.createStarted();
```

Comment:
question: Is this initialized here because there is no `operationStarted`? Do you know if there would be any delay between when this started and when the RPC is kicked off (I'm assuming that would be considered operation start)?

Or is it roughly the same because MetricsTracer is created when the RPC is kicked off?

Reply:
yes, the latter statement is true.